src/app.rs

```diff
@@ -47,6 +47,10 @@ impl Component for App {
     type Message = Msg;
     type Properties = ();
 
+    fn change(&mut self, _: <Self as yew::html::Component>::Properties) -> bool {
```

Comment:
```rust
fn change(&mut self, _props: Self::Properties) -> ShouldRender {
```
Could you update to this please ^ for consistency with update below? Also, I usually prefer `change` to be below `create`